### DISCUSSION TITLE
LVGL with ILI9488 driver

### DISCUSSION THREAD

**Discussion by garudaonekh** [arendst/Tasmota#15512] [Q&A] [answered]
Hi,
I see this guide https://tasmota.github.io/docs/LVGL/#try-a-lvgl-demo-app is based on ILI9341 driver.
I am using ILI9488 driver and different PIN mapping than the above guide.
 
What should I do to get it working with Tasmota?

Anyway, I can get my sample LVGL running in Arduino IDE.

Thanks;

**Comment by Jason2866** [comment, marked as answer]
Using this https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/ILI9488_display.ini
and replacing the current for ili9341 display descriptor should initialize your display.

**Reply by s-hadinger** [reply]
Don't forget to set one unused GPIO to "OptionA - 3". This enables the universal display driver.

**Reply by garudaonekh** [reply]
Can you share me link for detail on how to set the SPI PIN configuration?
